src/modules/endpoint/sagas/fetchResourceCache.js

```diff
@@ -40,15 +36,18 @@ export default function * fetchResourceCache (
     }
 
     const targetCategories: { [categoryPath: PathType]: Array<FilePathType> } =
-      invertBy(mapValues(fetchMap, ([url, path]) => path))
+      invertBy(mapValues(fetchMap, ([url, path, urlHash]) => path))
 
     const resourceCache = mapValues(targetCategories, filePaths =>
       reduce(filePaths, (acc, filePath) => {
         const downloadResult = successResults[filePath]
+        const [, , urlHash] = fetchMap[filePath]
```

Comment:
Maybe
```suggestion
        const urlHash = fetchMap[filePath][2]
```
would be clearer. But looks okay too